session/src/test/java/docs/javadsl/CassandraTestHelper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.javadsl;
+
+import akka.actor.ActorSystem;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import akka.stream.alpakka.cassandra.scaladsl.CassandraAccess;
+import akka.testkit.javadsl.TestKit;
+import com.datastax.oss.driver.api.core.CqlSession;
+
+import java.net.InetSocketAddress;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CassandraTestHelper {
+     ActorSystem system;
+     Materializer materializer;
+     CqlSession cqlSession;
+     CassandraAccess cassandraAccess;
+     String keyspaceName;
+    final  AtomicInteger tableNumber = new AtomicInteger();
+
+    public CassandraTestHelper(String TEST_NAME) {
+        system = ActorSystem.create(TEST_NAME);
+        materializer = ActorMaterializer.create(system);
+        cqlSession = CqlSession
+                .builder()
+                .withLocalDatacenter("datacenter1")
+                .addContactPoint(new InetSocketAddress("127.0.0.1", 9042))
+                .build();
```

Comment:
I think it would be better to use a CassandraSession also from the TestHelper, so that things are configured in the same way